File: ques.java
import java.util.ArrayList;

public class ques {
    public static int[] solution(int[] arr){
        int len=arr.length-1;
        Boolean[] val = new Boolean[len];
        ArrayList<Integer> arr_new = new ArrayList<Integer>();
        for (int i=0;i<=len;i++){
            int v=arr[i];
            if(val[v]){
                for(int j=0;j<=len;j++){
                    if(arr[j]==v){
                        arr[j].remove;
                    }

                }
            else{
                
            }
            }
        }
    }
    public static void main(String[] args){
        int[] arr={6,1,6,3,3};
        System.out.println("" +solution(arr));
    }
}

File: ques.py
def solution(arr):
    l=len(arr)-1
    val = [False] * 10
    for i in range(l):
        v = arr[i]
        if(val[v]):
            for j in range(l):
                if(arr[j]==v):
                    arr.remove(v)
                    l-=l
                    j=0
        else:
            val[v]= True 
    return arr

arr = [6,1,6,6,3,3]
solution(arr)
print(arr)